source/_docs/logs.md

```diff
@@ -112,7 +112,41 @@ You now have a local copy of the logs directory, which contains the following:
     └──mysqld-slow-query.log
     └──mysqld.log
 ```
-To automate this process, see [Automate Downloading Logs from the Live Environment](/docs/download-logs/).
+
+## Automate Downloading Logs
+
+You can automate the process of accessing and maintaining these logs with a simple script.
+
+### Enable Passwordless Access
+Logs are stored within application containers that house your site's codebase and files. [Add an SSH key](/docs/ssh-keys/) within your User Dashboard to enable passwordless access and avoid authentication prompts. Otherwise, provide your Pantheon Dashboard credentials when prompted.
+
+### Create Script
+Open terminal and run the following commands to create and access a new local directory:
+```
```

Comment:
Suggest adding bash to the three code snippets below for improved syntax highlighting (```bash)